### DISCUSSION TITLE
Generalize EZlink-scraper

### DISCUSSION THREAD

**Discussion by RubberArchind** [ZFC-Digital/puppeteer-real-browser#219] [General]
Did you have plan to make a general version for https://github.com/ZFC-Digital/ezlink-scraper? It will helpfull i think in many case
and will be a good start to make something like https://github.com/thomasdondorf/puppeteer-cluster?tab=readme-ov-file right?

@mdervisaygan

**Comment by mdervisaygan** [comment]
Hello, that project was a task requested in a recruitment process. For this reason, I didn't think of publishing it as a package, but I will check. Thank you.
